wagtail/project_template/project_name/settings/base.py

```diff
@@ -52,7 +52,7 @@
     'django.contrib.staticfiles',
 ]
 
-MIDDLEWARE_CLASSES = [
+MIDDLEWARE = [
```

Comment:
I don't think it's supposed to: https://github.com/torchbox/wagtail/blob/master/wagtail/project_template/requirements.txt#L1